modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/search_bar_configuration_suggestions/contributor_input_set_item/SiteActivities.js

```diff
@@ -73,14 +146,38 @@ function SiteActivities({index, onBlur, onInputSetItemChange, touched, value}) {
 		});
 	};
 
+	const _handleChangeAttributeValue = (property) => (newValue) => {
```

Comment:
To keep these 2 functions distinct, let's rename the other function `_handleChangeAttribute` to `_handleChangeAttributeInput`.